### PR TITLE
changeBG on stage

### PR DESCRIPTION
**Pull request by suhjainadobe**
Mergong changebg block on stage

Resolves: [MWPW-NUMBER](https://jira.corp.adobe.com/browse/MWPW-NUMBER)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/?martech=off
- After: https://interactivestagenew--cc--suhjainadobe.hlx.live/drafts/suhjain/imarquee?martech=off

### CODE REVIEW

**Review by aishwaryamathuria — APPROVED** [review]

**Comment by Ruchika4** [comment]
@suhjainadobe can we add hlx.live urls in PR description so that psi check can run?
Also I believe you had some unit tests too for ChangeBG, should we sync them also to stage branch?

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 1]
```diff
@@ -71,6 +71,11 @@ export default async function init(el) {
   const { decorateButtons, decorateBlockBg } = await import(`${miloLibs}/utils/decorate.js`);
   const { createTag, loadStyle } = await import(`${miloLibs}/utils/utils.js`);
   switch (true) {
+    case el.classList.contains('changebg'): {
+      const { default: changeBg } = await import('../../features/changeBg/changeBg.js');
+      changeBg(el);
```
Comment (aishwaryamathuria) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:76]: @suhjainadobe The PR intended for dexter consumption will need the changes to dynamically load JS and CSS from config.origin (if set) or else from relative domain. Could you make the required changes for the same?

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 2]
```diff
@@ -71,6 +71,11 @@ export default async function init(el) {
...
+      break;
+    }
     case el.classList.contains('genfill'): {
       loadStyle('/creativecloud/blocks/interactive-marquee/milo-marquee.css');
```
Comment (aishwaryamathuria) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:80]: @suhjainadobe The PR intended for dexter consumption will need the changes to dynamically load JS and CSS from config.origin (if set) or else from relative domain. Could you make the required changes for the same?

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 3]
```diff
@@ -71,6 +71,11 @@ export default async function init(el) {
...
       loadStyle('/creativecloud/features/genfill/genfill-interactive.css');
```
Comment (aishwaryamathuria) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:81]: @suhjainadobe The PR intended for dexter consumption will need the changes to dynamically load JS and CSS from config.origin (if set) or else from relative domain. Could you make the required changes for the same?

**Review by aishwaryamathuria — CHANGES_REQUESTED** [review]

**Comment by suhjainadobe** [comment]
As discussed... Will make the changes after testing with the original set.. Merging this change for initial validation.

**Pull request merged**